string opCode
string fileName
string code
---
string rsp